docs/config.md

```diff
@@ -72,7 +72,7 @@ The exporter is used to output the telemetry.
 
 | Environment variable | Description | Default |
 |-|-|-|
-| `OTEL_TRACES_EXPORTER` | The traces exporter to be used. Available values are: `zipkin`, `jeager`, `otlp`, `none`. | `otlp` |
+| `OTEL_TRACES_EXPORTER` | The traces exporter to be used. Available values are: `zipkin`, `jaeger`, `otlp`, `none`. | `otlp` |
```

Comment:
Can more than one exporter be defined? If that's the case, are values comma-separated, or it's a list object?

Reply:
No.  Only one value can be provided.
Maybe 

```suggestion
| `OTEL_TRACES_EXPORTER` | The traces exporter to be used. The value can be one of: `zipkin`, `jaeger`, `otlp`, `none`. | `otlp` |
```